packages/showcases/core/csf/components/VtmnTooltip.csf.js

```diff
@@ -0,0 +1,55 @@
+export const argTypes = {
+  position: {
+    type: { name: 'string', required: false },
+    description: 'The tooltip position',
+    defaultValue: 'top',
+    control: {
+      type: 'select',
+      options: [
+        'bottom-left',
+        'bottom',
+        'bottom-right',
+        'bottom',
+        'left',
+        'right',
+        'top-left',
+        'top',
+        'top-right',
+      ],
+    },
+  },
+
+  tooltip: {
+    type: { name: 'string', required: false },
```

Comment:
I think this field should be required, shouldn't it ?

Reply:
You're right !